### PR TITLE
WAVE2-924 Password prompt not working consistently with user roles when pages are password protected

### PR DESCRIPTION
**Pull request by gsowrey**
drupal_goto might be having issues with the protected pages params; basically short-cut all the URL setup for something more direct

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_core/ucgy_core_protected_pages/ucgy_core_protected_pages.module [hunk 1]
```diff
@@ -130,7 +130,11 @@ function ucgy_core_protected_pages_page_build() {
       $query['back'] = urlencode($current_path);
     }
     $query['ucgy_core_protected_page'] = $pid;
-    drupal_goto('protected-page', array('query' => $query));
+    //drupal_goto('protected-page', array('query' => $query)); This has periodic issues with the params...
+
+    $url = "http://ucalgary/protected-page?destination=" . $current_path . "&ucgy_core_protected_page=" . $pid;
```
Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_protected_pages/ucgy_core_protected_pages.module]: http://ucalgary.ca

Comment (dpfitzsi) [inline sites/all/modules/ucalgary/ucgy_core/ucgy_core_protected_pages/ucgy_core_protected_pages.module]: This should work for live environment, my only concern is about this working for others in this current state. We could pass in the ENV paramater here.

**Comment by gsowrey** [comment]
*facepalm

Stupid move, Geoff.

Good catch, Dan! Thanks!

**Comment by dpfitzsi** [comment]
Geoff is this tied to ticket 924 in Jira? "Password prompt not working consistently with user roles when pages are password protected"

**Comment by gsowrey** [comment]
Yep. Sorry, blew it on the PR name.

**Comment by dpfitzsi** [comment]
No problem. Just want to make sure I catch this when merging! PR has been renamed to reflect that.

**Pull request merged**